.github/actions/spelling/expect.txt

```diff
@@ -502,3 +566,16 @@ yao
 Zanollo
 ZIPHASH
 zoro
+clsid
+endregion
+ests
+Linq
+nameof
+netfx
+notmatch
+ools
+paramref
+Templating
+typeparam
+Unk
+websites
```

Comment:
Nit: can these be sorted?

Reply:
These will go away once it merges, they should be covered by one of the other dictionaries or some other part of the upgrade.